miniapps/nurbs/nurbs_naca_cmesh.cpp

```diff
@@ -0,0 +1,792 @@
+// Compile with: make nurbs_naca_cmesh
+//
+// Sample run:   ./nurbs_naca_cmesh -ntail 80 -nrad 80 -ntip 20 -nwake 40 -sw 2.0
+//                 -srad 2.5 -stip 1.1 -aoa 3
+//
+// Description:  This example code demonstrates the use of MFEM to create a C-mesh
+//               around a NACA-foil section. The foil section is defined in the class
+//               NACA4, which can be easily replaced with any other description of a
+//               foil section. To apply an angle of attack, we rotate the domain
+//               around the origin.
+//
+//               The mesh employs five patches of which two describe the domain behind the
+//               foil section (wake). The boundary describing the foil section is divided
+//               over three patches. One patch describes the domain adjacent to the boundary
+//               which describes the tip / leading edge of the foil section and two patches
+//               describe the domain which is adjacent to the two boundaries describing the
+//               remainder of the foil section. The aim is to create a mesh with the highest
+//               quality close to the boundary of the foil section and the wake.
+//
+//               The example returns a visit data structure for visualisation. Note that
```

Comment:
I'm not sure whether MFEM has a policy on correct vs. British spellings, e.g. `visualization` vs. `visualisation`. I suppose we can allow the British spelling, but it would be more consistent with the rest of the library to use `visualization`.

Reply:
I ran a spell checker with American English.